<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShardPage</class>
 <widget class="QWizardPage" name="ShardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Shards</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="shardList"/>
   </item>
   <item>
    <widget class="QGroupBox" name="shardInfo">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Information</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="usageLabel">
        <property name="text">
         <string>Usage</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QProgressBar" name="shardUsage">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="iPLabel">
        <property name="text">
         <string>IP</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="shardIP">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>shardList</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>shardInfo</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
